updatecli/scripts/update-calico.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
typically bash scripts also add "-eu" to make sure that all variables are set when they are called. It's a good practice to prevent unexpected executions

Reply:
I can add it to avoid possible errors on the script.